### PR TITLE
refactor: bab UI elements

### PR DESCRIPTION
**Pull request by PhearZero**
# Overview

- adds csv download for results
- cleans up UI elements and verbiage
- removes category from filter

### CODE REVIEW

**Review by HashMapsData2Value — COMMENTED** [review]
I noticed that proposals don't show the ask a

mount despite the .csv file requiring it, is this expected behavior?

File: src/build-a-bull/src/root.tsx [hunk 1]
```diff
@@ -27,14 +27,14 @@ export default function Root() {
...
   const heroRef = useRef<Element | null>(null)
-  const [offset, setOffset] = useState(0)
-  useEffect(() => {
-    const heroHeight = heroRef.current?.clientHeight || heroRef.current?.scrollHeight || 0
-    const headerHeight = headerRef.current?.clientHeight || headerRef.current?.scrollHeight || 0
-    if (typeof heroHeight === 'number' && typeof headerRef.current?.clientHeight === 'number') {
-      setOffset(heroHeight + headerHeight)
-    }
-  }, [headerRef, heroRef, setOffset])
+  // const [offset, setOffset] = useState(0)
```
[the remark this answers is not in the document]
Comment (PhearZero) [reply src/build-a-bull/src/root.tsx:30]: Exactly, it's possible we will use it

**Comment by HashMapsData2Value** [comment]
> > I noticed that proposals don't show the ask amount despite the .csv file requiring it, is this expected behavior?
> 
> Since it's a reskin of x-gov, the refactoring was limited to prevent too many bug regressions. Threshold/Ask are deeply integrated at this point.
> 
> We can do a proper refactor into reusable components/library if we end up continuing on this voting platform(TBD)

Okay, makes sense.

**Comment by HashMapsData2Value** [comment]
Sadly Im encountering the same issues I had before with HTTP 400, so I can't try voting. But I am able to close a vote and then download the results as a proper CSV.

So LGTM

**Pull request merged**